### PR TITLE
fix: close global backdrop

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-2623

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by guanbinrui** [comment]
Please link a JIRA issue.

File: packages/mask/src/UIRoot-page.tsx [hunk 1]
```diff
@@ -34,13 +34,13 @@ function MaskUIRoot({ children, useTheme, fallback }: MaskUIRootProps) {
...
     return compose(
-        (children) => DialogStackingProvider({ children, hasGlobalBackdrop: true }),
+        (children) => DialogStackingProvider({ children, hasGlobalBackdrop: false }),
         (children) => MaskThemeProvider({ useMaskIconPalette: (theme) => theme.palette.mode, useTheme, children }),
         (children) => Web3ContextProvider({ value: context, children }),
         (children) => SharedContextProvider({ children }),
         <Suspense fallback={fallback}>
             <CSSVariableInjector />
-            <GlobalBackdrop />
+            {/* <GlobalBackdrop /> */}
```
Comment (guanbinrui) [inline packages/mask/src/UIRoot-page.tsx]:
```suggestion
```

File: packages/mask/src/utils/shadow-root/renderInShadowRoot.tsx [hunk 2]
```diff
@@ -30,7 +29,7 @@ export function setupReactShadowRootEnvironment() {
     // Inject variable for Portals
     attachReactTreeToGlobalContainer(shadow, { key: 'css-vars' }).render(
         <>
-            <GlobalBackdrop />
+            {/* <GlobalBackdrop /> */}
```
Comment (guanbinrui) [inline packages/mask/src/utils/shadow-root/renderInShadowRoot.tsx]:
```suggestion
```

**Review by guanbinrui — APPROVED** [review]

File: packages/shared/src/contexts/components/InjectedDialog.tsx [hunk 3]
```diff
@@ -228,12 +228,18 @@ function CopyElementWithNewProps<T>(
...
+            const allKeys = new Set([...Object.keys(extraClasses as any), ...Object.keys(child?.props?.classes ?? {})])
+            const result: Record<string, string> = {}
+
+            for (const key of allKeys) {
+                result[key] = cx((extraClasses as any)[key], child?.props?.classes?.[key])
+            }
+            return child?.type === Target
                 ? cloneElement(child, {
-                      classes: cx(extraClasses as any, child.props.classes),
+                      classes: result,
```
Comment (Jack-Works) [inline packages/shared/src/contexts/components/InjectedDialog.tsx:240]: @nuanyang233 did you merge those classes?

**Pull request merged**